### PR TITLE
[ja] Added kubernetes-cluster-architecture.svg in content/ja/docs/concepts…

### PR DESCRIPTION
**Pull request by utkarsh-singh1**
Description:
Updated /docs/ja/concepts/architecture/_index.md and added kubernetes-cluster-architecture.svg in /static/images/docs.

Page to Update:
https://kubernetes.io/docs/ja/concepts/architecture/

### CODE REVIEW

**Comment by utkarsh-singh1** [comment]
Hi @sftim, @t-inu, Please review this PR.

**Comment by sftim** [comment]
We shouldn't make this change, because we're adding a new English diagram and changing Japanese content.

SIG Docs strongly prefers PRs that can be reviewed by reviewers who speak a single language.

**Comment by t-inu** [comment]
/hold
Feel free to unhold when PR #42272 is merged.

**Comment by t-inu** [comment]
The SVG file has been added in #42272, so it no longer needs to be included in this PR.

File: content/ja/docs/concepts/architecture/_index.md [hunk 1]
```diff
@@ -2,3 +2,5 @@
 title: "クラスターのアーキテクチャ"
 weight: 30
 ---
+
+{{< figure src="/images/docs/kubernetes-cluster-architecture.svg" alt="Kubernetesのコンポーネント" caption="kubernetes クラスターのアーキテクチャ" class="diagram-large" >}}
```
Comment (t-inu) [inline content/ja/docs/concepts/architecture/_index.md:6]:
```suggestion
{{< figure src="/images/docs/kubernetes-cluster-architecture.svg" alt="Kubernetesのコンポーネント" caption="kubernetesクラスターのアーキテクチャ" class="diagram-large" >}}
```

**Comment by t-inu** [comment]
Hi @utkarsh-singh1, please take a look at my comments about SVG file and caption.

**Comment by t-inu** [comment]
We will continue this work at #44917. Thank you.
/close

**Comment by utkarsh-singh1** [comment]
Sorry @t-inu, was busy with other issues.

**Pull request closed without merging**